Automattic/newspack-plugin
# Dashboard: remove tiers, switch to simple two-dimensional grid.

### All Submissions:






### Changes proposed in this Pull Request:



This PR removes the `Tier` component and switches the dashboard to a simple grid. 

Closes https://github.com/Automattic/newspack-plugin/issues/183.

### How to test the changes in this Pull Request:

1. `npm ci && npm run clean && npm run build:webpack`
2. Navigate to the Dashboard and observe the layout changes.

### Other information:
